### ISSUE TITLE
releases are not tagged

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#25]
```
Releases don't seem to be tagged in the SVN repository currently, so it is not 
possible to check out a specific version and build it. Would be good to tag 
versions every time you do a release. Or did I miss something?
```

Original issue reported on code.google.com by `eelco.hi...@gmail.com` on 30 Dec 2009 at 6:07

**Comment by GoogleCodeExporter** [comment]
Original comment by `manuel.carrasco.m` on 4 Jan 2010 at 10:53
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Hello, you are right, it's a good practice tag releases. 
I'll take care to do in each release, right now I've changed my deploy script 
to do 
it.

Fixed with release 0.5.6, 

Thank you for reporting.
```

Original comment by `manuel.carrasco.m` on 18 Feb 2010 at 5:27
- Changed state: **Fixed**
